templates/corporate/hello.html

```diff
@@ -1,562 +1,385 @@
-{% extends "zerver/portico.html" %}
-{% set entrypoint = "landing-page" %}
-
-{% set PAGE_TITLE = "Zulip: Open-source team chat with topic-based threading" %}
-
-{% set PAGE_DESCRIPTION = "Zulip is the only modern team chat app that is
-  designed for both live and asynchronous conversations. Organized to help you
-  collaborate productively and efficiently." %}
-
+{% extends "zerver/base.html" %}
+{% set entrypoint = "landing-page-hello" %}
 {% block customhead %}
-<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
-<style>
-    .portico-page {
-    padding-bottom: 0px;
-    }
-</style>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
+    <title>Zulip — organized team chat</title>
+    <!-- Google / search engine tags -->
+    <meta name="description" content="Zulip is the only modern team chat app that is designed for both live and asynchronous conversations. Organized to help you collaborate productively and efficiently." />
+    <!-- Open Graph / Facebook / Twitter meta tags -->
+    <meta property="og:url" content="https://zulip.com/" />
+    <meta property="og:type" content="website" />
+    <meta property="og:site_name" content="Zulip" />
+    <meta property="og:title" content="Zulip: Open-source team chat with topic-based threading" />
+    <meta property="og:description" content="Zulip is the only modern team chat app that is designed for both live and asynchronous conversations. Organized to help you collaborate productively and efficiently." />
+    <meta property="og:image" content="https://zulip.com/static/images/logo/zulip-icon-128x128.png" />
+    <meta name="twitter:card" content="summary" />
+    <!--   favicon -->
+    <link rel="icon" href="{{ static('images/landing-page/hello/favicon.ico') }}" sizes="any"/> <!-- 32×32 -->
+    <link rel="apple-touch-icon" href="{{ static('images/landing-page/hello/apple-touch-icon.png') }}"/> <!-- 180×180 -->
+    {% set entrypoint = entrypoint|default("portico") %}
```

Comment:
Is this a duplicate of `zerver/portico-header.html` or `templates/zerver/meta_tags.html`? IT would be nice to not duplicate those -- what's the thinking on duplicating rather than using that existing variable-based infrastructure?